### DISCUSSION TITLE
custom field to hold total power of a rack

### DISCUSSION THREAD

**Discussion by MohamedAYACH** [netbox-community/ansible_modules#747] [Q&A]
Hello,

I am new to Netbox, I added a custom field to hold the sum of all devices power in a rack, how to make this field compute the desired value automatically ?

Thank's in advance.

**Comment by sc68cal** [comment]
NetBox Rack objects already have fields for maximum power in a rack as well as computing the power utilization for you, based on the devices in the rack. You will need to ensure that all your devices have power consumption values for this to work.
